examples/powertools-examples-core/sam/src/main/java/helloworld/App.java

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Any idea why these are marked as new files? 
Has anything changed - do we need to diff this externally?